### PR TITLE
Clickability

### PR DESCRIPTION
**Pull request by shaymarieee**
seems like theres a conflict or two, but yeah i got my style images rendering correctly!

### CODE REVIEW

File: client/src/components/Overview/ImageCarousel.jsx [hunk 1]
```diff
@@ -6,11 +6,11 @@ import React from 'react';
 // }
 
 const ImageCarousel = (props) => {
-  {if (props.currentStyle.photos) {
-    {console.log('FOR THUMBIES', props.currentStyle.photos)}
+  {if (props.productImageCarousel !== []) {
+    {console.log('ooohhhh', props)}
```
Comment (Jcase22) [inline client/src/components/Overview/ImageCarousel.jsx]: left a console.log here

File: client/src/components/Overview/OverviewIndex.jsx [hunk 2]
```diff
@@ -9,32 +9,44 @@ import StyleSelector from './StyleSelector.jsx';
...
         setProductStyles(styles.data.results);
-        styles.data.results.forEach((style) => {
+        let hasDefault = false; // will this work?
+        styles.data.results.map((style, index) => {
           if (style.['default?']) {
+            hasDefault = true;
             setProductImage(style.photos[0].url || 'http://placecorgi.com/260/180')
             setCurrentStyle(style)
-          }
+            setProductImageCarousel(style.photos)
```
Comment (Jcase22) [inline client/src/components/Overview/OverviewIndex.jsx:26]: Nice way of handling the case where there is no default

**Review by Jcase22 — CHANGES_REQUESTED** [review]
Looks good!

**Review by Jcase22 — APPROVED** [review]
looks good!

File: client/src/components/Overview/ImageCarousel.jsx (hunk 1 above)
Comment (shaymarieee) [reply client/src/components/Overview/ImageCarousel.jsx]: dangit i thought i got em all, thanks!! ill take it out and push again in a sec (:

**Pull request merged**